app/src/main/java/com/hsLink/hslink/presentation/onboarding/component/screen/LinkScreen.kt

```
@@ -0,0 +1,231 @@
+            .removePrefix("www.")
+
+        val domain = withoutProtocol.split("/")[0]
+
+        when {
+            domain.contains("instagram.com") -> "인스타그램"
+            domain.contains("github.com") -> "깃허브"
+            domain.contains("linkedin.com") -> "링크드인"
+            domain.contains("notion.so") || domain.contains("notion.site") -> "노션"
+            else -> domain.take(30)
+        }
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)